src/KeepAlive.ts

```diff
@@ -96,9 +104,11 @@ export class KeepAlive extends EventEmitter {
 
   /**
    * Start monitoring the RTMClient. This method should only be called after the client's websocket is already open.
-   * @param client
+   * @param client an RTMClient to monitor
    */
   public start(client: RTMClient): void {
+    this.logger.debug('start monitoring');
```

Comment:
Should we make these a little less vague so they make more sense in the context of all the other debug logging an app will be doing?

Reply:
i think this is okay since each log line is prepended with the object name (e.g. `KeepAlive`), and in that context, monitoring only really has one meaning.